fv/run-batches

```diff
@@ -29,8 +29,7 @@ for batch in ${FV_BATCHES_TO_RUN}; do
                -ginkgo.seed 1 \
                -ginkgo.randomizeAllSpecs=true \
                -ginkgo.slowSpecThreshold ${FV_SLOW_SPEC_THRESH} \
-               -ginkgo.focus="${GINKGO_FOCUS}" \
-               ${GINKGO_ARGS}
```

Comment:
What is this change for?

Reply:
${GINKGO_ARGS} is set to "-mod=vendor" which isn't a valid option for the compiled fv.test

Reply:
${GINKGO_ARGS} is set to "-mod=vendor" which isn't a valid option for the compiled fv.test